src/main/java/viper/Instruction.java

```diff
@@ -0,0 +1,29 @@
+package viper;
+
+public enum Instruction {
+    LIST("list"),
+    DONE("done"),
+    TODO("todo"),
+    EVENT("event"),
+    DEADLINE("deadline"),
+    DELETE("delete"),
+    BYE("bye"),
+    INVALID("invalid"),
+    EMPTY("empty"),
+    ;
+
+    private final String s;
```

Comment:
Should this variable have a more descriptive name to let the user know what this variable is storing?